### PR TITLE
Add class addBook - Week 1 - Module 2

### PR DESCRIPTION
**Pull request by fabianomag**
Replace all objects for classes

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -6,19 +6,18 @@ const booksContainer = document.querySelector('.books-container');
...
-
-function addBook(title, author) {
-  const book = {
-    title,
-    author,
-    id: parseInt((Math.random() * 1000000000), 10),
-  };
-  booksCollection.push(book);
-  localStorage.setItem('books', JSON.stringify(booksCollection));
+class AddBook {
```
Comment (rukundoeric) [inline script.js:9]: - [ ] <b>REQUIRED:</b> `AddBook` should not be a class. it should be a function.

File: script.js [hunk 2]
```diff
@@ -6,19 +6,18 @@ const booksContainer = document.querySelector('.books-container');
...
 }
 
 function handleChange() {
-  addBook(bookTitle.value, bookAuthor.value);
+  const book = new AddBook(bookTitle.value, bookAuthor.value);
+  booksCollection.push(book);
+  localStorage.setItem('this', JSON.stringify(booksCollection));
 }
 
 function removeBook() {
```
Comment (rukundoeric) [inline script.js:23]: - [ ] <b>REQUIRED:</b> And all functions should be inside `Book` class. Kindly do that to comply with the project requirements. here is an example:
```js
class Books {
  constructor() {
    this.books = [];
  }

  addBook(title, author) {
  }

  removeBook(bookID) {
  }

  // getter method
  getBooks() {
  }
}

```

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @fabianofrank, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [x] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

**Hi team**,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**